### DISCUSSION TITLE
orjson in alpine lxc

### DISCUSSION THREAD

**Discussion by chinoosniel** [django-commons/django-unicorn#198] [Q&A]
Although maybe I am off topic a bit, but I've been trying to use "orjson" inside an LXC container with ALPINE LINUX for a while. But so far I have not had any luck, I wonder if there is any way to achieve it. Or to use django-unicorn inside a LXC ALPINE container.
Thank you very much for this wonderful project.

**Comment by adamghill** [comment]
This might not be what you were hoping for, but it appears that wheels can't be used in Alpine, so `orjson` can't be installed: https://github.com/ijl/orjson/issues/98#issuecomment-642833810.

Do you think you could use a different Docker image?

I could also try to use a different JSON serializer if `orjson` isn't available although I'm a little worried about the additional complexity and the performance implications of not using `orjson`.

What do you think is the best way forward?

**Reply by adamghill** [reply]
Just investigated more issues on the `orjson` repo and _maybe_ it is possible with the right Docker incantation?

Long thread with some possible solutions: https://github.com/ijl/orjson/issues/8
